src/main/java/com/Elessar/app/server/RegisterHandler.java

```diff
@@ -49,22 +52,50 @@ public void handle(HttpExchange he) throws IOException {
             final RegistrationRequest regRequest = RegistrationRequest.parseFrom(is);
             final RegistrationResponse.Builder regResponse = RegistrationResponse.newBuilder();
             final String userName = regRequest.getName();
+            final User existingUser = users.get(userName);
```

Comment:
why use get() here and other places getUnchecked()?

Reply:
I changed it for debug purpose and forgot to change it back..